src/components/personalinformation/PersonalInfoItem.js

```diff
@@ -18,7 +18,10 @@ const PersonalInfoItem = ({ label, value, bold, preview, onclick, showIcon = fal
   return (
     <div className="personal-info-row">
       <div className="personal-info-row__item">
-        <div className="personal-info-row__label">Your {label}</div>
+        <div className="personal-info-row__label">
+          {preview ? '' : 'Your '}
```

Comment:
can we use template literals?

Reply:
I fear it'd look uglier still. This is what I could come up with.

> {`${preview ? '' : 'Your'} ${capitalize(label)}`}

Reply:
If you have a better way please suggest.

Reply:
{preview?'':`Your ${capitalize(label)}` }

Reply:
This would hide the label in Preview mode. Also we will have to redesign the preview page since the place where label existed would be empty now. I'd rather keep the labels in preview also because it's clearer.